scripts/check_urls/check_urls.ts

```diff
@@ -0,0 +1,57 @@
+import fs from "fs";
+import YAML from "yaml";
+
+interface UrlObject {
+  description: string;
+  tags: string[];
+  title: string;
+  url: string;
+}
+const filePath = "../../data/resources.yml";
+
+fs.readFile(filePath, "utf-8", (err, data) => {
+  const urls: UrlObject[] = YAML.parse(data);
+
+  const checkUrl = async (url: UrlObject) => {
+    try {
+      const response = await fetch(new URL(url.url).toString(), {
+        method: "HEAD",
+      });
+      if (response.status == 200 && !response.redirected) {
+        return { url: url.url, status: response.status };
+      } else if (response.redirected) {
+        return {
+          url: url.url,
+          status: 301,
+          redirect: response.url,
+        };
+      } else {
+        return { url: url.url, status: response.status };
+      }
+    } catch (err) {
+      console.error(err);
+      return { url: url.url, status: "Error" };
+    }
+  };
+
+  Promise.all(urls.map(checkUrl)).then((links) => {
+    links
+      .filter((l) => l.status == 200)
```

Comment:
All `2XX` status codes are all ok. It would be good to futureproof this in case we get non 200 passing codes.